lms/djangoapps/experiments/utils.py

```diff
@@ -50,7 +50,14 @@ def get_experiment_user_metadata_context(course, user):
     enrollment_mode = None
     enrollment_time = None
     enrollment = None
+    # TODO: clean up as part of REVO-18
+    has_paid_enrollments = None
     try:
+        # TODO: clean up as part of REVO-18 (START)
+        user_enrollments = CourseEnrollment.objects.select_related('course').filter(user_id=user.id)
+        audit_enrollments = user_enrollments.filter(mode='audit')
+        has_paid_enrollments = (len(audit_enrollments) != len(user_enrollments))
```

Comment:
there are non-paid enrollments that are not of mode audit, like honor.   
(and people could have a verified/professional/no-id-professional enrollment without having paid). these are both edge cases though that may not be relevant for the test.  
but I would probably still try to name the variables accurately, like has_non_audit_enrollments  instead of has_paid_enrollments for example